### PR TITLE
Update dr-ui@0.25.4

### PR DESCRIPTION
**Pull request by katydecorah**
This PR contains documentation infrastructure updates.

dr-ui@0.25.1: https://github.com/mapbox/dr-ui/blob/master/CHANGELOG.md#0251

## Infrastructure updates

* [x] Update Sentry, dr-ui, mr-ui, remark-lint-mapbox
* [x] Remove `babelInclude: ['mimic-fn', 'debounce-fn’],` line from batfish.config.js 
* [x] Use [prettier-config-docs](https://github.com/mapbox/prettier-config-docs#prettier-config-docs)
  - [x] Add/Replace `"prettier": "@mapbox/prettier-config-docs"` in package.json
  - [x] Delete `.prettierrc.json`
  - [x] In package.json > scripts add `"format": "prettier --write '**/*.{js,css,html,json}'",`
  - [x] run `npm run format && git commit -am "run prettier"`
* [x] update docs-page-shell

## Component updates

* `Note` https://mapbox.github.io/dr-ui/#note
    * [x] check Notes `themes`, make sure beta notes are using "beta" theme (some may be using "new") and use new `download` theme where appropriate.
    * [x] search for and remove all instances of `imageComponent`, `BookImage`,  `NewImage` and `WarningImage`
* `Tag` https://mapbox.github.io/dr-ui/#tag
    * [ ] Update OverviewHeader, NavigationAccordion, Product menu (where applicable) with new `tag` prop.
* `CodeSnippet` https://mapbox.github.io/dr-ui/#codesnippet
    * [x] Replace uses of mr-ui/code-snippet for dr-ui/code-snippet
    * [x] Replace inlined prism css instances with `@mapbox/dr-ui/highlight/theme-css.js`
* `Edit` https://mapbox.github.io/dr-ui/#edit
    * [x] (mapbox-gl-js only) replace current edit buttons with dr-ui version

## Testing
* [ ] Push to publisher-staging
* [x] Confirm site loads on IE 11 (locally and staging)
* [x] Review [dr-ui changelog](https://github.com/mapbox/dr-ui/blob/master/CHANGELOG.md#0251) to confirm no breaking changes since previous version
* [x] Check package.json for unused dependencies, move any deps to devDeps as needed
* [x] Run `npm audit` or `npm audit fix` to fix dependencies (where possible)
* [x] Run `npm install @mapbox/mbx-assembly@latest`

Ref https://github.com/mapbox/documentation/issues/352

### CODE REVIEW

**Comment by katydecorah** [comment]
Holding on review request until https://github.com/jsfiddle/jsfiddle-issues/issues/1524 is resolved

**Comment by katydecorah** [comment]
@colleenmcginnis we're ready! I didn't push to staging since we're using it in #189

**Comment by colleenmcginnis** [comment]
@katydecorah I'm seeing some weird spacing issues in some of the larger code blocks in the Style Spec. For example: 

Screen Shot 2020-01-22 at 4 39 27 PM

Screen Shot 2020-01-22 at 4 39 17 PM

**Comment by katydecorah** [comment]
@colleenmcginnis wow! Thanks for finding that. I just pushed a fix to add back some code that handles fixing those indents.

File: docs/pages/api.js [hunk 1]
```diff
@@ -92,15 +91,26 @@ export default class Api extends React.Component {
                         , a compatible renderer written in C++ with bindings for
                         desktop and mobile platforms.
                     </p>
-                    <DrUiNote
-                        title="Pricing for Mapbox GL JS"
-                        imageComponent={<BookImage />}
-                    >
+                    <DrUiNote title="Pricing for Mapbox GL JS" theme="warning">
```
Comment (colleenmcginnis) [inline docs/pages/api.js]: In a recent PR we changed this note from the warning theme to the default theme in production: https://docs.mapbox.com/mapbox-gl-js/api/

```suggestion
                    <DrUiNote title="Pricing for Mapbox GL JS">
```

**Pull request merged**